src/styles/stars.scss

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("../images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  min-width: 16px;
+  min-height: 16px;
+
+  &:not(:last-child) {
+    margin-right: 4px;
+  }
+}
+
+.stars--1 .stars__star:nth-child(-n+1),
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url("../images/star-active.svg");
```

Comment:
<img width="789" alt="Screenshot 2023-08-09 at 15 44 25" src="https://github.com/mate-academy/layout_catalog/assets/135232001/1f40cc66-479e-4e3f-82f1-7b4a4d87f6dd">
Consider refactoring your code and using SASS features